src/Microsoft.AspNet.OData.Shared/IDeltaSet.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// Basic interface to reperesenta deltaset which is a collection of Deltas.
+    /// </summary>
+    public interface IDeltaSet
```

Comment:
>IDeltaSet [](start = 21, length = 9)

Is this really just a marker interface? I would think that IDeltaSet : ICollection<IDeltaSetItem>, and include the public Patch method.  Ideally we could pass the interface to consuming methods, rather than the concrete type.

Reply:
+1

---
In reply to: [616255482](https://github.com/OData/WebApi/pull/2455#discussion_r616255482) [](ancestors = 616255482)

Reply:
Patch Method is typed right? DeltaSet<TStructuralType> Patch(IPatchMethodHandler patchHandler)